libcxx/test/std/iterators/predef.iterators/reverse.iterators/reverse.iter.elem/arrow.pass.cpp

```diff
@@ -24,6 +24,54 @@
 
 #include "test_macros.h"
 
+#if TEST_STD_VER >= 20
+template <class It>
+class cpp20_bidirectional_iterator_with_arrow {
```

Comment:
```suggestion
// C++20 bidirectional_iterator that does not satisfy the Cpp17BidirectionalIterator named requirement.
template <class It>
class cpp20_bidirectional_iterator_with_arrow {
```

That way, it's less likely that a future refactoring will accidentally remove the test coverage by making this a Cpp17BidirectionalIterator.